cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/BuiltInRolesAdministrationTest.scala

```diff
@@ -0,0 +1,329 @@
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+
+        p("The resulting `architect` role has the same privileges as the " +
+          "<<administration-built-in-roles-architect-privileges,original built-in `architect` role>>.")
+
+//        query("SHOW ROLE architect PRIVILEGES", ResultAssertions(p => true)) {
+//          resultTable()
+//        }
+      }
+    }
+    section("The `admin` role", "administration-built-in-roles-admin", "enterprise-edition") {
+      p("The `admin` can do the same as  the <<administration-built-in-roles-architect,`architect`>>, as well as manage databases, users, roles and privileges.")
+      section("Privileges of the `admin` role", "administration-built-in-roles-admin-privileges", "enterprise-edition") {
+        query("SHOW ROLE admin PRIVILEGES",ResultAssertions(p => true)) {
+          resultTable()
+        }
+      }
+      section("How to recreate the `admin` role", "administration-built-in-roles-admin-recreate", "enterprise-edition") {
+        initQueries(
+          "DROP ROLE admin")  // setup so that later when the grant query gets executed, it will show systemUpdates: 1
+        p(
+          """
+            |To restore the role to its original capabilities two steps are needed: First, if not already done, execute `DROP ROLE admin`.
+            |Secondly, the following queries need to be executed in order to set up the privileges:""".stripMargin)
+        query("CREATE ROLE admin", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT ALL DBMS PRIVILEGES ON DBMS TO admin", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT TRANSACTION MANAGEMENT ON DATABASE * TO admin", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT START ON DATABASE * TO admin", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT STOP ON DATABASE * TO admin", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT MATCH {*} ON GRAPH * TO admin", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 2)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT WRITE ON GRAPH * TO admin", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 2)
+        })) {
+          statsOnlyResultTable()
+        }
+        query("GRANT ALL ON DATABASE * TO admin", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          statsOnlyResultTable()
+        }
```

Comment:
I think the order here is a bit messy. I would add those commands that are part of earlier roles first. Also I'm thinking about if it would be more pedagogical to split the `ALL ON DATABASE` into its sub commands like we are doing for the publisher.

Reply:
There is a reason why the sub privileges are used for the publisher: After the queries, I wanted to have the exact same output of SHOW ROLE x PRIVILEGES from before. This is not possible for the admin role, so that's why I could go with the compound commands (so that it would be fewer queries to execute). When it comes to the order, I don't think it matters much and I wanted the keep the same order as in the operations manual because it describes the same process.

Reply:
Ok that sounds reasonable